test/lib/miq/md_link_converter_test.rb

```diff
@@ -25,4 +25,11 @@ def test_multiline_conversion
 
     assert_equal expected, Miq::MdLinkConverter.new(original).convert
   end
+
+  def test_converting_external_md_href
+    original = "[Title](https://github.com/ManageIQ/ui-components/blob/master/README.md)"
+    expected = "[Title](https://github.com/ManageIQ/ui-components/blob/master/README.md)"
+
+    assert_equal expected, Miq::MdLinkConverter.new(original).convert
+  end
```

Comment:
Can you add tests for the manageiq.org cases?

Reply:
addressed